static/style.css

```diff
@@ -0,0 +1,794 @@
+
+.info {
+  --olp: 24px;
+  --bp: 114px;
+
+  display: flex;
+  flex-flow: column;
+  gap: 12px;
+  padding: var(--bp) var(--olp);
+  width: 100%;
+}
+
+.intro {
+  background: url(../images/header-shapes-mobile.svg), url(../images/header-background-mobile.svg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: right top, bottom left;
+}
+
+.socialmedia-btn-row {
+  display: flex;
+  gap: 20px;
+}
+
+.social-btn {
+  display: inline-block;
+  height: 20px;
+  width: 20px;
+  border: 0;
+}
+
+.social-btn:hover {
+  background-color: var(--border);
+  box-shadow: 0 8px 16px var(--body-bg);
+}
+
+.social-btn:active {
+  background: var(--btn-active);
+}
+
+#btn-m {
+  height: 16px;
+}
+
+#btn-gh {
+  width: 18px;
+}
+
+#btn-pc {
+  width: 14.17px;
+}
+
+/* **** Works Section **** */
+
+.works {
+  --w: 704px;
+  --n: 2;
+
+  grid-template-columns: repeat(auto-fit, minmax(clamp(100%/(var(--n) + 1) + 0.1%, (var(--w) - 100vw) * 1000, 100%), 1fr));
+  padding: 114px 24px;
+  display: grid;
+  column-gap: 24px;
+  row-gap: 88px;
+  justify-content: center;
+}
+
+.works img {
+  width: 100%;
+  height: auto;
+}
+
+.work {
+  padding: 16px;
+  gap: 12px;
+  align-items: center;
+  border: 0.1px solid var(--border);
+  border-radius: 16px;
+  display: flex;
+  background-color: var(--bckgrd);
+  flex-flow: column;
+  flex-grow: 0;
+}
+
+.work-card {
+  display: flex;
+  flex-flow: column;
+  gap: 20px;
+}
+
+.work-card li {
+  display: flex;
+  gap: 12px;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.work-btn {
+  color: var(--btn-color);
+  background-color: var(--bckgrd);
+  border: 1px solid var(--font-2);
+  border-radius: 8px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  cursor: pointer;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 141%;
+  letter-spacing: 0.03em;
+  z-index: 1;
+}
+
+.work-card .work-btn {
+  margin-block: 8px;
+}
+
+.work-cat {
+  display: flex;
+  gap: 12px;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.cat-title {
+  font-weight: 600;
+  color: var(--font-3);
+  font-size: 13px;
+  line-height: 123%;
+}
+
+.cat-descrip {
+  background: url(../images/Counter.svg) no-repeat;
+  background-position: left center;
+  font-weight: 600;
+  color: var(--font-3);
+  font-size: 13px;
+  line-height: 123%;
+  padding-left: 16px;
+}
+
+.work-descrip {
+  font-weight: 400;
+  color: var(--font-3);
+  font-size: 15px;
+  line-height: 160%;
+}
+
+.work-langs {
+  display: flex;
+  gap: 8px;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+}
+
+.work-lang {
+  padding: 4px 12px;
+  background: var(--connect);
+  border-radius: 8px;
+}
+
+.work-lang span {
+  font-weight: 500;
+  color: var(black);
+  font-size: 12px;
+  line-height: 133%;
+}
+
+/* **** Button interacticity **** */
+
+.work-btn:hover {
+  background: var(--font-2);
+  color: var(--bckgrd);
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+}
```

Comment:
[OPTIONAL] Please stick to the Figma design I think the previous reviewers missed this requirement your hover effect is not the same as the Figma design so kindly fix this issue this requirement is optional but recommended so kindly don't skip this requirement :slightly_smiling_face: 

![Screenshot from 2023-02-17 17-31-04](https://user-images.githubusercontent.com/45226848/230715339-ebaba8ca-38ec-4527-b1b2-0fa9003b267f.png)